_data/students.yml

```diff
@@ -188,3 +188,10 @@
   image: Krish.jpg
   twitter: krishbirla13
   
+- name: Arjun Pandey
+  country: India
+  github: arjunpandey2003
+  image: arjun.jpg
+  twitter: arjunpandey2003
+  lat: 77.2090
+  lng: 28.6139 
```

Comment:
leave a single empty line at the end of the file. Otherwise LGTM

Reply:
its already there